### ISSUE TITLE
error when mastodon account is following a lot of accounts already

### ISSUE THREAD

**Issue by bunnyhero** [WesSouza/mastodon-flock#6]
i get an error at the "mastodon.exe" part of the setup after i authorize my octodon.social account.

steps to reproduce:

1. start mastodon flock setup
2. authorize twitter
3. choose "typical" setup
4. enter "octodon.social" as instance url
5. authorize octodon.social
6. watch progress (twitter.exe, "searching mastodon followers" mastodon.exe)

result:
- "there was an unknown error" windows-style dialog
- "details" button just closes the wizard

expected result:
- successful setup

i've been able to run mastodon flock successfully with a different account on a different instance, using the same steps. i get an error on octodon only.

thanks!

**Comment by WesSouza** [comment]
I wonder if this is a JavaScript error on our end that I never encountered while testing with my accounts.

Octodon seems to be closed for sign ups, so I'd need your help to debug.

I have installed an error tracker on the preview environment, but you can also open the developer tools (right click on the page and select Inspect), open the "Console" tab, and check if any red messages appear, then either paste a screenshot or the error message.

Can you retry on that environment?
https://mastodon-flock-preview.vercel.app/

**Comment by bunnyhero** [comment]
i tried with the preview environment and i see this error in the console:

```
Object { code: "504", message: "An error occurred with your deployment" }
```

**Comment by WesSouza** [comment]
I think I found the problem, it's taking too long to read the list of accounts you follow and timing out the serverless request.

I'm working on a fix.

**Comment by WesSouza** [comment]
@bunnyhero one more time, please:
https://mastodon-flock-preview.vercel.app/

**Comment by bunnyhero** [comment]
yesss that fixes it. thank you!

**Comment by WesSouza** [comment]
Fixed by a8bf94a132a4ef0126efa42c2fab24aa8fbe58b4.
